requirements.txt

```diff
@@ -1,7 +1,7 @@
 ninja
 flaky
 numpy
-pennylane>=0.15
+pennylane~=0.26
```

Comment:
```suggestion
pennylane~=0.25
```

Reply:
Pip will not pick up on dev or rc releases nicely, so best to keep this fixed to the most current release.

Reply:
We can revisit this once 0.25 goes live.

Reply:
Thank you! I intended it to be 0.25